src/KeysManager.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\pubkey_encrypt\KeysManager.
+ */
+
+namespace Drupal\pubkey_encrypt;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\user\PrivateTempStoreFactory;
+use Drupal\user\UserInterface;
+
+/**
+ * Handles users' Public/Private key pairs.
+ */
+class KeysManager {
```

Comment:
Agreed that this name is too generic. But wouldn't calling the class EncryptionManager might make some people think that it is responsible for providing the main encryption decryption functionality which it actually is not, since we'll be leveraging Field Encrypt for that. 

What are your thoughts on calling this class UserKeysManager and putting it inside pubkey_encrypt namespace. This would make the complete service name as: pubkey_encrypt.user_keys_manager. 

Let me know if you still want me to proceed with the PublicKeyEncryptionManager and I will.